web/js/components/ReadSupers.jsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import Superlative from './Superlative.jsx'
+import Notifications, {notify} from 'react-notify-toast';
+
+// Hard-coding supers for testing 
+var supers = [
+  {
+  "name":"Most likely to die in a freak accident",
+  "created_by":"A",
+  "assigned_by":"B",
+  "receipient":"C"
+  },
+  {
+  "name":"Most loyal",
+  "created_by":"D",
+  "assigned_by":"E",
+  "receipient":"F"
+  },
+  {
+  "name":"Best hair",
+  "created_by":"D",
+  "assigned_by":"E",
+  "receipient":"F"
+  },
+  {
+  "name":"Most likely to become president",
+  "created_by":"D",
+  "assigned_by":"E",
+  "receipient":"F"
+  }
+]
+
+export default class ReadSupers extends React.Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      supersList:supers
+    }
+
+    this.nextButtonClicked = this.nextButtonClicked.bind(this)
+
+  }
+  render() {
+    return (
+      <div>
+        <Notifications />
+        <Superlative name={this.state.supersList[0].name}/>
+        <button onClick={this.nextButtonClicked}>Next superlative</button>
+      </div>
+    )
+  }
+
+  // Load supers from server when the DOM is loaded
+  componentDidMount(){
+    this.setState({
+      supersList:supers,
+    })
+  }
+
+  nextButtonClicked(){
+    
+    // Save assignment to server
+    // If there are supers left, go to next super; otherwise go to Done screen.
+    this.state.supersList.shift()
```

Comment:
It's generally considered bad practice to mutate state directly.

you can use 
```this.setState({
    supersList: supersList.slice(1)
})```

and then remove the this.setState(this.state) below


Then instead of the 
```
this.props.onStatusChange(true,"That's it! Host a new game or join an existing one.");
```

You should have your `render()` function be aware of the list being empty and display that message if the supersList is empty.